check_py_coverage.sh

```diff
@@ -91,7 +91,8 @@ cd ../PostProcessors/TopologicalPostProcessor
 coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_topology_ui.xml interactiveCheck
 cd ../DataMiningPostProcessor/Clustering/
 coverage run $EXTRA $FRAMEWORK_DIR/Driver.py hierarchical_ui.xml interactiveCheck
-cd ../../../../../
 
+## Go to the final directory and generate the html documents
+cd $SCRIPT_DIR/tests/framework
```

Comment:
can we add a pwd after this cd, just so we know where we are (in case it doesn't work)?